java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsBaseCommands.java

```diff
@@ -0,0 +1,46 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Scripting and Function" group for standalone and
+ * cluster clients.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/?group=scripting">Scripting and Function
+ *     Commands</a>
+ */
+public interface ScriptingAndFunctionsBaseCommands {
+
+    /**
+     * Loads a library to Redis.<br>
+     * When in cluster mode, this command is routed to all primary nodes.
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/function-load/">redis.io</a> for details.
+     * @param libraryCode The source code that implements the library.
+     * @return The library name that was loaded.
+     * @example
+     *     <pre>{@code
+     * String code = "#!lua name=mylib \n redis.register_function('myfunc', function(keys, args) return args[1] end)";
+     * String response = client.functionLoad(code).get();
+     * assert response.equals("mylib");
+     * }</pre>
+     */
+    CompletableFuture<String> functionLoad(String libraryCode);
```

Comment:
Does it make sense to follow how we implemented `Script` objects but for `FunctionCode`? 
We Load scripts into a single Node, and hash it on the client-side so that if it needs to be called against a different node it can be loaded as-needed/ad-hoc.

Reply:
No, answer from aws team:
> Because functions were developed to be gossiped to other nodes in the server side, unlike script that is under the client responsibility to handle failovers by reloading the script
See https://redis.io/docs/latest/develop/interact/programmability/functions-intro/